### PR TITLE
fix: k8s bitnami repo index error

### PR DESCRIPTION
**Pull request by zhongjiajie**
ref: apache/dolphinscheduler#10567

### CODE REVIEW

**Comment by zhongjiajie** [comment]
PTAL @caishunfeng @liqingwang

**Review by caishunfeng — APPROVED** [review]
+1

File: docs/1.3.5/docs/zh/kubernetes-deployment.md [hunk 1]
```diff
@@ -8,6 +8,12 @@
 
 ## 安装 dolphinscheduler
 
+> 注意：您需要在本地更改 `Chart.yaml` 文件才能使其正常工作。 由于 Bitnami 存储库的更改，https://charts.bitnami.com/bitnami 被截断，
+> 仅包含最近 6 个月（从 2022 年 1 月起）的条目。 只有这个 url 才包含了：https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami 完整的 index.yaml。
+> 如果你想要了解更多详细信息，请查看：https://github.com/bitnami/charts/issues/10833。
```
[the remark this answers is not in the document]
Comment (zhongjiajie) [reply docs/1.3.5/docs/zh/kubernetes-deployment.md:13]: Great, and will change it right now

File: docs/1.3.5/docs/en/kubernetes-deployment.md [hunk 2]
```diff
@@ -8,6 +8,14 @@
...
 ## Installing the Chart
 
+> NOTE: You need to change your `Chart.yaml` locally to make it work. Due to a change in the Bitnami repo,
+> https://charts.bitnami.com/bitnami was truncated only containing entries for the latest 6 months (from January 2022 on).
+> This URL: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami contains the full 'index.yaml'.
+> See detail here: https://github.com/bitnami/charts/issues/10833.
+>
+> Change you `Chart.yaml` and replace all `repository: https://charts.bitnami.com/bitnami`
+> to `repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami`.
+
```
[the remark this answers is not in the document]
Comment (liqingwang) [reply docs/1.3.5/docs/en/kubernetes-deployment.md:20]: Wouldn't it be better to modify the Chinese document in this way? WDYT?

File: docs/1.3.5/docs/en/kubernetes-deployment.md [hunk 3]
```diff
@@ -8,6 +8,16 @@
 
 ## Installing the Chart
 
+> NOTE: You need to change your `Chart.yaml` locally to make it work. Due to a change in the Bitnami repo,
+> https://charts.bitnami.com/bitnami was truncated only containing entries for the latest 6 months (from January 2022 on).
+> This URL: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami contains the full 'index.yaml'.
+>
+> See detail here: https://github.com/bitnami/charts/issues/10833.
+>
+> Change you `Chart.yaml` in path `apache-dolphinscheduler-incubating-1.3.5-src-release/docker/kubernetes/dolphinscheduler` after you download from GitHub
```
Comment (liqingwang) [inline docs/1.3.5/docs/en/kubernetes-deployment.md:17]: `after you download from GitHub` 

Is the download link of the official documentation not apache?

**Comment by zhongjiajie** [comment]
Thanks all

**Pull request merged**